tests/test_alignments.py

```diff
@@ -0,0 +1,37 @@
+import os
+import shutil
+
+from fixtures import DataDir, en_sample, ru_sample
+
+current_folder = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Question: We have been a bit inconsistent with UPPER_CASE and lower_case variables at the root level. Do you have a preference?

Reply:
usually upper case is for constants and lower case is for variables